plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/ElasticsearchQueryRunner.java

```diff
@@ -91,14 +92,16 @@ public static Builder builder(ElasticsearchServer server)
     public static final class Builder
             extends DistributedQueryRunner.Builder<Builder>
     {
+        private static final AtomicLong instanceId = new AtomicLong();
         private final ElasticsearchServer server;
         private final Map<String, String> connectorProperties = new HashMap<>();
         private List<TpchTable<?>> initialTables = ImmutableList.of();
 
         private Builder(ElasticsearchServer server)
         {
             super(testSessionBuilder()
-                    .setCatalog("elasticsearch")
+                    // We need separate catalog names because these tests check for JMX beans that would clash otherwise
+                    .setCatalog("elasticsearch_" + instanceId.incrementAndGet())
```

Comment:
This is a problem of a test, not of a query runner.

ElasticsearchQueryRunner should, just like every other query runner, configure catalog and schema names statically. 

Many other plugins do expose mbeans. Why is this problem specific to ES?

Reply:
Did you try configuring `jmx.base-name` in a particular test that accesses these mbeans?

Reply:
No but this could work too